pygeoapi/templates/_base.html

```diff
@@ -20,8 +21,14 @@
       <link rel="{{ link['rel'] }}" type="{{ link['type'] }}" title="{{ link['title'] }}" href="{{ link['href'] }}"/>
       {% endif %}
     {% endfor %}
+    <link rel="shortcut icon" href="{{ config['server']['url'] }}/static/img/favicon.ico" type="image/x-icon">
+    <link rel="stylesheet" href="https://unpkg.com/mini.css@3.0.1/dist/mini-default.min.css">
```

Comment:
Aren't we using bootstrap now (instead of mini.css)? https://github.com/geopython/pygeoapi/blob/master/pygeoapi/templates/_base.html

Reply:
Indeed, must be a problem in the rebase, ll check